ocrd/ocrd/decorators.py

```diff
@@ -27,9 +32,7 @@ def ocrd_cli_wrap_processor(processorClass, ocrd_tool=None, mets=None, working_d
         print(msg)
         raise Exception(msg)
     else:
-        if mets.find('://') == -1:
-            mets = 'file://' + os.path.abspath(mets)
-        if mets.startswith('file://') and not os.path.exists(mets[len('file://'):]):
+        if is_local_filename(mets) and not isfile(get_local_filename(mets)):
             msg = "File does not exist: %s" % mets
             print(msg)
```

Comment:
And here?

Reply:
The message is printed (so it appears in STDOUT) and an exception raised. What else would you expect?

Reply:
`LOG.error` ?

Reply:
done.